Server/db/models/StatusPage.js

```diff
@@ -0,0 +1,42 @@
+import mongoose from "mongoose";
+
+const StatusPageSchema = mongoose.Schema(
+	{
+		companyName: {
+			type: String,
+			required: true,
+			default: "",
+		},
+		url: {
+			type: String,
+			unique: true,
+			required: true,
+			default: "",
+		},
+		timezone: {
+			type: String,
+			required: true,
+			default: "America/Toronto",
+		},
+		color: {
+			type: String,
+			required: true,
+			default: "#4169E1",
+		},
+		theme: {
+			type: String,
+			required: true,
+			default: "light",
+		},
```

Comment:
Do we need to store the theme as well? I thought that was dealt with locally

Reply:
@marcelluscaio yes I'm not quite sure of the usage of this one either, I just followed the spec and there was a `theme` field in it :stuck_out_tongue:  It could be that the spec was from an earlier iteration before we had theme defined at all actually.

If it turns out we don't need this we can remove it when we're clear on its usage :+1:   Thanks for being vigilant!